arrows/klv/tests/test_klv_0601.cxx

```diff
@@ -168,7 +168,7 @@ auto const expected_result = klv_local_set{
       KLV_0102_COUNTRY_CODING_METHOD_GENC_THREE_LETTER,
       std::string{ "CAN" }, kv::nullopt, std::string{ "FRA" } } },
   { KLV_0601_NUMBER_OF_NAVSATS_IN_VIEW,        uint64_t{ 7 } },
-  { KLV_0601_POSITIONING_METHOD_SOURCE,        klv_blob{ { 0x03 } } },
+  { KLV_0601_POSITIONING_METHOD_SOURCE,        uint64_t{ 3 } },
```

Comment:
I would suggest making this more explicit (using `1 << ENUM | ...`), to match the `KLV_0601_GENERIC_FLAG_DATA` entry on lines 78-81.